.github/workflows/cicd-release-validation.yml

```diff
@@ -0,0 +1,580 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# This is the top-level workflow that runs on release branches.
+# It invokes other jobs to perform builds and run tests.
+# All jobs run in parallel, using build artifacts to synchronize jobs.
+#
+# If you add or remove any tests that use reusable-test.yml on a pull request,
+# you must update codecov.yml to match.
+
+name: CI/CD - Release validation
+
+on:
+  # Permit manual runs of the workflow.
+  workflow_dispatch:
+    inputs:
+      # Request a target release branch to run CI/CD on.
+      branch:
+        description: 'Target release branch to run CI/CD on.'
+        required: true
+        default: 'release/'
+
+concurrency:
+  # Cancel any in-progress instance of this workflow (CI/CD) for the same PR.
+  # Allow running concurrently with any commits on any other branch.
+  # Using GITHUB_REF instead of GITHUB_SHA allows parallel runs on
+  # different branches with the same HEAD commit.
+  group: cicd-release-validation-${{ github.event.schedule || github.event.after || github.event.pull_request.number || github.ref }}
```

Comment:
For this workflow, you don't need these parameters. (and can remove the comment above)
"cicd-release-validation" would actually be enough, or can put this:
`group: ${{ github.workflow }}-${{ github.ref }}`

Reply:
Updated

Reply:
DId you push an update? Line 28 looks as before

Reply:
no, not yet, was waiting for other comments...